cassandra_check/check.py

```diff
@@ -0,0 +1,78 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import re
+import shlex
+
+# project
+from checks import AgentCheck
+from utils.subprocess_output import get_subprocess_output
+from collections import defaultdict
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'cassandra_check'
+DEFAULT_HOST = 'localhost'
+DEFAULT_PORT = '7199'
+
+class CassandraCheck(AgentCheck):
+
+    datacenter_name_re = re.compile('^Datacenter: (.*)')
+    host_status_re = re.compile('^(?P<status>[UD])[NLJM].* (?P<owns>(\d+\.\d+%)|\?).*')
```

Comment:
since we're here, why not parsing all the info nodetool returns? 
Specifically:
 * Status (gauge)
 * State (tag)
 * Address (tag)
 * Load (gauge)
 * Tokens (we can probably ignore, this param is not easy to change on a cluster)
 * Owns (gauge)
 * Host ID (tag)
 * Rack (tag, this would be the availability zone on EC2)